tests/bearlib/languages/documentation/documentation_extraction_testdata/default.py

```diff
@@ -29,7 +25,6 @@ def get_55():
 """ Docstring inline with triple quotes.
```

Comment:
Note that this, and the one above, are *not* a docstring.  They are incorrectly described as such (https://github.com/coala/coala/issues/4566), and shouldnt be parsed as such (is this being done?).

Also my understanding of https://www.python.org/dev/peps/pep-0008/#blank-lines is that two blank lines are expected after the prior def, and before the next def.

```bash
$ pycodestyle default.py
...
default.py.1:28:1: E302 expected 2 blank lines, found 1
default.py.1:42:1: E302 expected 2 blank lines, found 1
...
```